.github/workflows/build.yml

```diff
@@ -160,3 +160,29 @@ jobs:
       - uses: actions/upload-artifact@v2
         with:
           path: ./wheelhouse/*.whl
+  wheel-gpu:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-python@v2
+        name: Install Python
+        with:
+          python-version: '3.7'
+      - name: Install cibuildwheel
+        run: |
+          python -m pip install -U cibuildwheel==2.2.2
+      - name: Install terra
+        run: |
+          pip install -U qiskit-terra
```

Comment:
FWIW, this won't actually install terra in the right place. Since you're installing it before running cibuildwheel this installs terra in the host environment while cibuildwheel operates in a container which won't see this install. You'd have to do this as part of the the cibuildwheel options to get it to install.